packages/zipper/lib/zip-bundle/index.ts

```diff
@@ -94,10 +92,13 @@ export const zipBundle = async (
       if (aborted) return;
 
       const absPath = resolve(distDirectory, file);
-      const relPath = relative(distDirectory, absPath); // Get the relative path
+      const absPosixPath = posix.resolve(distDirectory, file);
+      const relPosixPath = posix.relative(distDirectory, absPosixPath);
+
+      console.log(`Adding file: ${relPosixPath}`);
       streamFileToZip(
         absPath,
-        relPath, // Use relative path
+        relPosixPath, // Use relative path
```

Comment:
```suggestion
        relPosixPath,
```

@govza This is your TODO, i think :)

Reply:
> It's not to-do but a comment, not mine
> 
> I'm leaving for holidays, so feel free to do what's necessary and merge 😉

I will do, next week.

Have a good time on holiday ❤